### PR TITLE
feat: port media/formats/* to cmake

### PR DESCRIPTION
**Pull request by cosmin**

### CODE REVIEW

File: packager/utils/absl_flag_hexbytes.cc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//
+// Created by Cosmin Stejerean on 7/18/23.
```
Comment (joeyparrish) [inline packager/utils/absl_flag_hexbytes.cc:2]: Please fix this to a copyright header consistent with the rest of the project.

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Finally reviewed.  Sorry it took so long to make time for this!

File: packager/media/formats/wvm/wvm_media_parser.cc [hunk 2]
```diff
@@ -87,6 +88,15 @@ namespace shaka {
 namespace media {
 namespace wvm {
 
+uint32_t ntohlFromBuffer(const void* buffer) {
```
[the remark this answers is not in the document]
Comment (cosmin) [reply packager/media/formats/wvm/wvm_media_parser.cc:91]: It should be used below in several places but I'll update these callsites to use Load32

File: packager/media/base/muxer.h [hunk 3]
```diff
@@ -56,7 +54,7 @@ class Muxer : public MediaHandler {
   /// If no clock is injected, the code uses std::chrone::system_clock::now()
   /// to generate the time-stamps.
   /// @param clock is the Clock to be injected.
-  void set_clock(MuxerClock clock) { clock_ = clock; }
+  void set_clock(Clock* clock) { clock_ = clock; }
```
[the remark this answers is not in the document]
Comment (cosmin) [reply packager/media/base/muxer.h]: Updated to use unique_ptr internally and keep passing in a raw pointer and reset to it for now. The caller of this method is going to be in the next pull request (in app). We can potentially change the calling code at that point to directly pass in a unique_ptr and use std::move.

**Review by joeyparrish — APPROVED** [review]

**Pull request merged**